app/server/__init__.py

```diff
@@ -123,8 +123,9 @@ def after_request(response):
 
         for task in g.celery_tasks:
             try:
+                # TODO: Standardize this task (pipe through execute_synchronous_celery)
```

Comment:
Note: I'm not immediately worried about this one, since I don't see anywhere that g.celery_tasks is actually written to

Reply:
yeah I put this in here for tasks that we only wanted to execute AFTER the server db had committed, to avoid a race condition where the task might request an object by its ID, only to find the object doesn't exist yet, but then at some point in a refactor found ourselves not actually needing it. Could come in useful in the future though, if we start returning more data from workers to the main server